anvil/src/winit.rs

```diff
@@ -193,7 +193,8 @@ pub fn run_winit(display: &mut Display, event_loop: &mut EventLoop<()>, log: Log
         {
             running.store(false, Ordering::SeqCst);
         } else {
-            display.flush_clients();
+            // FIXME: should we supply non-() data?
```

Comment:
No, currently anvil does not use the `DispatchData`, this can be left for a future refactor.